resources/DK.kreve

```diff
@@ -0,0 +1,75 @@
+################################################################################
```

Comment:
Same as above

Reply:
No - not a part of the final distribution, but necessary to share right now, to get KE synchronized

Reply:
Alright. Just wanted to make sure this don't get merged accidentally.